.github/workflows/issues.yml

```diff
@@ -22,6 +22,7 @@ jobs:
             const issues = await github.rest.issues.listForRepo({
               state: 'all',
               assignee: '*',
+              sort: 'updated',
```

Comment:
Won't this mean that if I continually make changes to one of my existing issues, I won't get assigned any new issues?

Reply:
Oh yeah you're right. I was thinking the issue coming in would be the one we'd want to look at. Duh.